src/app/backend/replicasetlist.go

```diff
@@ -98,6 +98,7 @@ func getReplicaSetList(
 	replicaSets []api.ReplicationController, services []api.Service) *ReplicaSetList {
 
 	replicaSetList := &ReplicaSetList{}
+	replicaSetList.ReplicaSets = make([]ReplicaSet, 0)
```

Comment:
Move this statement to initialization phase, i.e., `&ReplicaSetList{ReplicaSets = make([]ReplicaSet, 0)}`. Try to initialize in the init object whenever possible.